models/Media/Media.js

```diff
@@ -80,6 +80,28 @@ MediaSchema.statics.create = function (type, data) {
   });
 };
 
+MediaSchema.methods.getMedia = function (type) {
```

Comment:
What's the idea behind this method? In the endpoint to get poll, it doesn't use it but just uses `media.poll`
Looks like for images it wasn't updated, but couldn't we do something similar like `media.image.buffer.toString(`base64`)

I don't think it's a bad change, just wondering what the exact reasoning is

Reply:
The reasoning was because of the .image.buffer.toString, I thought it would be a better idea to centralize that in the Media schema. I just didn't update the poll side of things

Reply:
Yeah I do think realistically this is probably better, design-wise :+1:

Reply:
My opinion is that the more code we can hide in the schemas the cleaner the code will be (to an extent at least)

Reply:
Agreed :smile: